index-handlers/src/main/java/no/sikt/nva/nvi/index/aws/CandidateQuery.java

```diff
@@ -110,10 +111,12 @@ private static Query statusQuery(String customer, ApprovalStatus status) {
     }
 
     private static Query termQuery(String value, String field) {
-        return new TermQuery.Builder()
-            .value(new FieldValue.Builder().stringValue(value).build())
-            .field(field)
-            .build()._toQuery();
+        return nonNull(value)
```

Comment:
Is it possible to adjust the nvi admin search test to verify that this is fixing the bug?

Reply:
OpenSearchTest:
```
@Test
void shouldNotThrowExceptionWhenSearchingWithFilterWithoutInstitution() {
    var searchParameters = CandidateSearchParameters.builder()
                               .withUsername(randomString())
                               .withFilter("pending")
                               .build();
    assertDoesNotThrow(() -> openSearchClient.search(searchParameters));
}
```